PowerPointLabs/PowerPointLabs/ColorPicker/MagnifierForm.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Windows.Forms;
+
+using PPExtraEventHelper;
+
+namespace PowerPointLabs.ColorPicker
+{
+    public partial class MagnifierForm : Form
```

Comment:
Regarding changing from WinForm to WPF, can I do it in a separate PR? It lives in the backend since the form rendered content is replaced by Magnification API.

I tried using WPF but somehow Native.CreateWindowEx/Magnification API doesn't work nicely with WPF. Not sure if its due to window handle or what.